vm/src/stdlib/nt.rs

```diff
@@ -445,6 +445,39 @@ pub(crate) mod module {
         Ok(vm.ctx.new_list(drives))
     }
 
+    #[pyfunction]
+    fn listvolumes(vm: &VirtualMachine) -> PyResult<PyListRef> {
+        let mut volumes = vec![];
+        let mut buffer = [0u16; 257];
```

Comment:
If it is related to constant, is this possible?
```suggestion
        let mut buffer = [0u16; MAX_PATH + 1];
```